src/debug/gdb/gdb.c

```diff
@@ -290,8 +290,7 @@ static inline int gdb_parser(void)
 				mem_to_hex(sregs + addr, remcom_out_buffer, 4);
```

Comment:
@mh12gx2825 something is wrong with the first commit. You shouldn't send merge commits in PR.